framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java

```diff
@@ -500,19 +502,23 @@ public TestStatus verifyTextPlan(String expectedOutput,
     throw new VerificationException(sb.toString());
   }
 
-  private static boolean matchesAll(String actual, String expected) {
+  private static boolean matchesAll(String actual, String expected,
+                                    boolean noOrder) {
```

Comment:
Let's change the boolean name to checkOrder. We don't want to use double negative (!noOrder). Change it to 'if (checkOrder) {}